examples/wordpress/resources/app.yaml

```diff
@@ -0,0 +1,128 @@
+# Sample application manifest for Wordpress, the open source content management system
+
+apiVersion: cluster.gravitational.io/v2
+kind: Cluster
+metadata:
+  name: wordpress
+  resourceVersion: "0.0.1"
+
+# Note the use of Elastic Blockstore now available for Persistent Volumes since 7.0
+storage:
+ openebs:
+   enabled: true
+
+# The logo can be used to white label Gravity web interface for Kubernetes/Wordpress
+# management
+logo: https://bitnami.com/assets/stacks/wordpress/img/wordpress-stack-220x234.png
+
+# This section declares the endpoints Wordpress requires.
+endpoints:
+ - name: "wordpress"
+   description: "Wordpress Application"
+   selector:
+    name: wordpress
+   protocol: http
+
+# This section allows to customize the graphical (web UI) installer
+# for the cluster.
+installer:
+
+  # if 'flavors' section is present, the installer will ask the end user what
+  # type of Kubernetes cluster to create. We are offering either a small
+  # single node, 3 node or 5 node configuration.
+  flavors:
+    prompt: "What size install do you want (small, medium, or large)?"
+    # The default flavor
+    default: small
+    # List of flavors:
+    items:
+    - name: "small"
+      description: "Single node"
+      nodes:
+      - profile: general
+        count: 1
+    - name: "medium"
+      description: "Three node install with one Master, one DB worker node, and one Front worker node "
+      nodes:
+      - profile: master
+        count: 1
+      - profile: front
+        count: 1
+      - profile: db
+        count: 1
+#HA Master nodes (3) configuration with a 2 fronts and a db
+    - name: "large"
+      description: "High Availability Master Node configuration with 2 Wordpress PHP nodes and 1 DB node"
+      nodes:
+      - profile: front
+        count: 2
+      - profile: db
+        count: 1
+      - profile: master
+        count: 3
+
+# This section allows to define what flavors of servers are required for
+# this cluster. This example uses a single flavor called "node" but you
+# can have flavors like "database" or "gpu", etc.
```

Comment:
Ditto here - the flavors are `general`, `master`, `front` and `db`, not a single flavor `node`